ddtrace/tracer/tracer_test.go

```diff
@@ -1095,6 +1098,25 @@ func decode(p *payload) (spanLists, error) {
 	return traces, err
 }
 
+func (t *dummyTransport) waitFlush(ts *testing.T, n int) {
+	a := time.After(1 * time.Second)
```

Comment:
I think you misunderstood my previous comment. I just meant to call this variable `timeout` or `done`.

Reply:
I did! I'll rename the channel.